tests/Feature/GithubAuthenticationTest.php

```diff
@@ -94,6 +95,65 @@ public function test_redirect_on_login_fail()
         $this->assertEquals(0, User::count());
     }
 
+    public function test_user_is_created_with_default_user_role()
```

Comment:
This test can be merged with the one that already existed when you started working: `test_user_is_created_after_first_login`